### PR TITLE
Learning guide city design svg

### PR DESCRIPTION
**Pull request by darrenmason**
updated svg for chart design;  still need to update detail footer, etc.

image

### CODE REVIEW

**Comment by philschatz** [comment]
Great! Can you update the tests?

**Comment by darrenmason** [comment]
this commit contains all the changes from #129

**Comment by philschatz** [comment]
This PR is tied to openstax/tutor-server#153

**Comment by darrenmason** [comment]
image

File: style/resources/cityscape.svg [hunk 1]
```diff
@@ -0,0 +1,383 @@
+<svg
```
Comment (philschatz) [inline style/resources/cityscape.svg]: Is this file still used?

File: src/components/learning-guide-chart.coffee [hunk 2]
```diff
@@ -3,44 +3,57 @@ d3 = require 'd3'
 
 CLOUDS_PATH = '/style/resources/clouds.svg'
```
Comment (philschatz) [inline src/components/learning-guide-chart.coffee]: Also, do we still need this file (and var)?

File: style/resources/cityscape.svg (hunk 1 above)
Comment (darrenmason) [inline style/resources/cityscape.svg]: for the moment, I will replace it next

File: src/components/learning-guide-chart.coffee [hunk 3]
```diff
@@ -3,44 +3,57 @@ d3 = require 'd3'
...
 PLANE_PATH = '/style/resources/openstax-plane.svg'
+CITYSCAPE_PATH = '/style/resources/cityscape.svg'
+
+# should be svg bound to the label text, but temp for now.
+FLAG_BLUE = '/style/resources/flag-blue.png'
```
Comment (philschatz) [inline src/components/learning-guide-chart.coffee]: Any reason these aren't drawn as SVG or included (with text) in the flight path background? Maybe this could be a separate PR?

Comment (darrenmason) [inline src/components/learning-guide-chart.coffee]: the flags should be separate from the background so they can be positioned independently.  I don't have svg assets for the flags yet

File: src/components/learning-guide-chart.coffee (hunk 2 above)
Comment (darrenmason) [inline src/components/learning-guide-chart.coffee]: I will remove it.

File: style/resources/cityscape.svg (hunk 1 above)
Comment (darrenmason) [inline style/resources/cityscape.svg]: @Fredasaurus, regarding the new svg asset, I need 1 single separate cloud svg and 1 separate city svg.  The gradient and green bar are already in the chart, so do not need those.  Also if you want the colored flags to be svg I need those separate .svg files as well. (w/o the text)
